html/admin/project/project_assetsBoard.twig

```diff
@@ -49,6 +49,36 @@
         </div>
     </div>
 </div>
+{% if "PROJECTS:PROJECT_ASSETS:EDIT:ASSIGNMENT_STATUS"|instancePermissions %}
+<div class="modal fade" id="scanAssetModal">
+    <div class="modal-dialog modal-lg">
+        <div class="modal-content">
+            <div class="modal-header">
+                <h4 class="modal-title">Scan In Asset</h4>
+                <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+                    <span aria-hidden="true">&times;</span>
+                </button>
+            </div>
+            <div class="modal-body">
+                <p>Note: only assets from the currently selected business ({{ USERDATA.instance.instances_name }}) can be scanned.</p>
+                <div class="form-group assetStatusDiv">
+                    <label>Asset Status</label>
+                    <select class="form-control" id="assetStatus">
+                        {% for status in BOARDSTATUSES[USERDATA.instance["instances_id"]] %}
+                        <option value="{{ status['assetsAssignmentsStatus_id'] }}">{{ status['assetsAssignmentsStatus_name'] }}</option>
+                        {% endfor %}
+                    </select>
+                </div>
+                <div class="form-group assetTagDiv">
+                    <label>Asset Tag or Barcode</label>
```

Comment:
I think you're looking to just find assets by Asset Tag, so lets make this dialog just for asset tag actions and we can look to support barcodes separately.